cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinAffiliation.java

```diff
@@ -0,0 +1,41 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.util.Collections;
+import java.util.Map;
+import nva.commons.core.JacocoGenerated;
+
+@SuppressWarnings("unused")
+@JacocoGenerated
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CristinAffiliation {
+
+    private Boolean active;
+    private Map<String, String> position;
```

Comment:
It is not clear what a position is. A class could be helpful.

Reply:
But the problem is that we need dynamic field names (different language codes) and when using a map we can dynamically set these using keys

Reply:
Still not clear to me what is a position

Reply:
We discussed it and concluded we do not have a better way of creating dynamic field names for the moment, so now just renaming to a more descriptive value